base/loggers/Loggers.h

```diff
@@ -41,6 +41,6 @@ class Loggers
 
     std::weak_ptr<DB::TextLog> text_log;
     int text_log_max_priority = -1;
-
+    int logger_level;
```

Comment:
Why do we need to store this log level? Looks like we take it from the config each time.

Reply:
It's unnecessary to reset the log level when they are the same.

https://github.com/ClickHouse/ClickHouse/pull/18401/files#diff-43a52ce116732798748d49ce68e6e20f9cd1c47027327c635cefc7c2e870c69dR198